Staggered upgrade procedure for OVS clouds (#1408)
stackhpc/stackhpc-kayobe-config: doc/source/operations/upgrading-openstack.rst

   kayobe overcloud service upgrade --tags config --kolla-tags keystone

OVS
^^^

You should first stop the Octavia health manager to prevent alerts during
the service upgrade.

.. code-block:: console

   kayobe overcloud host command run --command "docker stop octavia_health_manager" --limit controllers --become

For dedicated network nodes, upgrade the control plane services:

.. code-block:: console

>    kayobe overcloud serivce upgrade --kolla-limit controllers

Reviewer: ```suggestion
   kayobe overcloud service upgrade --kolla-limit controllers
```

For converged network nodes, you should specify the service limit to only
upgrade the Neutron API service.

.. code-block:: console

>    kayobe overcloud serivce upgrade --kolla-limit controllers -ke neutron_service_limit=neutron-server

Reviewer: ```suggestion
   kayobe overcloud service upgrade --kolla-limit controllers -ke neutron_service_limit=neutron-server
```

> To ensure L3 reliability during the upgrade, we will need to manually drain

Reviewer: ```suggestion
To ensure L3 reliability during the upgrade, we will need to sequentially drain
and upgrade each network node by first disabling agents and then running a targeted upgrade.
```
